app/controllers/settings/users/index.js

```diff
@@ -145,4 +128,12 @@ export default class UsersSettingsController extends Controller {
     }
     await Promise.all(this.membershipsBeingBlocked.map((membership) => membership.save()));
   }
+
+  @action
+  filterMemberships(memberships) {
+    return memberships.filter((membership) => {
+      const shouldShow = !this.showBlockedMemberships || membership.get('isBlocked');
```

Comment:
I got confused between `showBlockedMemberships` and `showBlockMemberships`. Can we rename the tracked properties to something more distinctive like `showBlockedMembershipsOnly` (for the filter checkbox) and `showBlockMembershipsConfirmationModal` (for the modal flag)?
Same for the other tracked properties in this controller.